section11/models/user.py

```diff
@@ -12,12 +12,6 @@ def __init__(self, username, password):
         self.username = username
         self.password = password
 
-    def json(self):
-        return {
-            'id': self.id,
-            'username': self.username
-        }
-
```

Comment:
I noticed that it is not included originally, but I feel like it can be useful when we are introducing blacklisting. Since we may need to see the `id` of the user, and explain that he would be blacklisted.

Reply:
But we don't have any endpoint that would allow us to see it.

Reply:
Ahh, I forgot. I added a User resource to do that in the advanced course code. Another fix would be to return the user JSON when successfully registered a new user. Which one do you prefer? I can make the changes now.

Reply:
It may be best to have the User resource just for completeness. Let's merge these and add that in in a separate PR